src/App.tsx

```diff
@@ -1,25 +1,40 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
-
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42).map(n => `Item ${n}`);
+import { Pagination } from './components/Pagination';
 
 export const App: React.FC = () => {
+  const items = getNumbers(1, 42).map(n => `Item ${n}`);
```

Comment:
It's not a good practice to generate data directly in the component. It's better to move this logic to a separate function or file. This will make your component more readable and maintainable.